src/main/java/com/sergiofah/integracao/controller/ProductPageController.java

```diff
@@ -16,12 +18,17 @@
 import javafx.scene.image.Image;
 import javafx.scene.image.ImageView;
 import javafx.scene.layout.AnchorPane;
+import javafx.scene.paint.Color;
+
 
 public class ProductPageController {
-	
-	private String selectedLine;
 
-	private ProductDAO productDAO;
+	private Long selectedLineId;
+
+	private Service service;
+
+	@FXML
+	private Label serverStatus;
 
 	Image loading;
```

Comment:
By our team code convention, it's necessary to declare the `@FXML` attributes first.